packages/nibijs-examples/01_new-wallet.ts

```diff
@@ -0,0 +1,16 @@
+import { newRandomWallet } from "@nibiruchain/nibijs"
+import { logSuccess } from "./shared"
+
+async function runExample() {
```

Comment:
Use es6 arrow functions in examples

Ex. `const runExample = async () => {...}`

Reply:
fixed